packages/design-system-docs/src/pages/components/Table/table.example.html

```diff
@@ -2,7 +2,7 @@
 <% [ 
   { title: 'Default table', className: 'ds-c-table'}, 
   { title: 'Borderless table', className: 'ds-c-table--borderless'}, 
-  { title: 'Striped table', className: 'ds-c-table--striped'}, 
+  { title: 'Striped and densed table', className: 'ds-c-table--striped ds-c-table--dense'}, 
```

Comment:
I would recommend splitting these examples up, so 1 dense table and striped table